client/dataHelper.js

```diff
@@ -69,70 +69,79 @@ export async function sendUpdatedPassword (token, oldPass, newPass) {
  */
 export async function getAllPhrases(token) {
     const getPhrases = await fetch("/api/getPhrases", {
         method: "POST",
         headers: {
-            "Content-Type": "application/json"
+            "Content-Type": "application/json",
         },
-        body: JSON.stringify({"token": token})
-    })
-    if(getPhrases.status === 400 || getPhrases.status === 401 || getPhrases.status === 500) {
-        window.alert((await getPhrases.text()).toString())
-    } else if(getPhrases.status === 200) {
-        return JSON.parse(await getPhrases.text())
+        body: JSON.stringify({ token: token }),
+    });
+    if (
+        getPhrases.status === 400 ||
+        getPhrases.status === 401 ||
+        getPhrases.status === 500
+    ) {
+        window.alert((await getPhrases.text()).toString());
+    } else if (getPhrases.status === 200) {
+        return JSON.parse(await getPhrases.text());
     }
 }
 
 /**
  * Retrieves a list of valid stock image file names from the database
- * 
+ *
  * @param {String} token The session token from the cookie system
  * @returns {Array} A list of file names as an array
  */
 export async function getAllStockImages(token) {
     const getStockImages = await fetch("/api/getStockImages", {
         method: "POST",
         headers: {
-            "Content-Type": "application/json"
+            "Content-Type": "application/json",
         },
-        body: JSON.stringify({"token": token})
-    })
-    if(getStockImages.status === 400 || getStockImages.status === 401 || getStockImages.status === 500) {
-        window.alert((await getStockImages.text()).toString())
-    } else if(getStockImages.status === 200) {
-        return JSON.parse(await getStockImages.text())
+        body: JSON.stringify({ token: token }),
+    });
+    if (
+        getStockImages.status === 400 ||
+        getStockImages.status === 401 ||
+        getStockImages.status === 500
+    ) {
+        window.alert((await getStockImages.text()).toString());
+    } else if (getStockImages.status === 200) {
+        return JSON.parse(await getStockImages.text());
     }
 }
 
 /**
  * Creates a new post from the content passed into it
- * 
+ *
  * @param {Object} newPost JSON object consisting of the session token, post content, and picture file name
  * @returns {int} Response code from the API
  */
 export async function createPost(newPost) {
     const response = await fetch("/api/post", {
         method: "POST",
         headers: {
-            "Content-Type": "application/json"
+            "Content-Type": "application/json",
         },
-        body: JSON.stringify(newPost)
-    })
-    
-    if(response.status === 401 || response.status === 500) {
-        window.alert((await response.text()).toString())
-        setIsFormDisabled(false)
-    }
-    else if(response.status === 200) {
-        window.alert("Post created successfully!\nPost id: " + (await response.text()))
+        body: JSON.stringify(newPost),
+    });
+
```

Comment:
Change this to respond only to code 200 and show alert window for all other codes